README.md

```diff
@@ -2,3 +2,20 @@
 A browser extension to add VTOL back to bonk.io
 
 [Discord link](https://discord.gg/Dj6usq7ww3)
+
+# installation instructions
+This plugin is distributed as a firefox plugin.
```

Comment:
It's an extension and it works on Chromium-based browsers too. For now you should add a subtitle called "Firefox", so me or someone else can add instructions for Chromium later